recipes/license_server.rb

```diff
@@ -65,6 +71,11 @@
 
 b = ruby_block 'license cleanup' do
   block do
+    existing_directory = Dir.glob("#{node['splunk']['home']}/etc/licenses/*")
+    existing_directory.each do |dir|
+      FileUtils.rm_rf(dir) unless dir.end_with?(type)
```

Comment:
Alrighty now just need to fix the logic a bit... as we want to only delete those directories that we expect there to be no licenses, so something like: (double check me on this... this is rather haphazard, but should give you the general idea)

```ruby
next if license_groups.keys.include? File.basename(dir)
Chef::Log.info( "... blah..." )
FileUtils.rm_rf(dir)
```

Reply:
This should be fixed with d4c6810